packages/frontend/app/styles/components/programs/list.scss

```diff
@@ -2,4 +2,8 @@
 
 .list {
   @include m.main-list-box-table;
+
+  &.empty {
```

Comment:
You put this in the right place to work, but this style is coming from the wrong file.

It's outside the scope of this PR but this `programs/list.scss` style should be scoped down to only target the program list  and not every `.list` in the application (everything should be under a `.programs-list{` class). Can you make that change here or circle back and make a PR just to do that and then re-do the style in this one? It will also require creating a component style for the learner group list (and possibly other lists which are also accidentally replying on this style).

Reply:
Let's chat about this at our next meeting.